mne_qt_browser/_pg_figure.py

```diff
@@ -42,6 +42,7 @@
     QPoint,
     QSettings,
     QSignalBlocker,
+    QRegExp,
```

Comment:
Instead of `QRegExp` and `QRegExpValidator`, if you want a regular expression based validator, please use `QRegularExpression` and `QRegularExpressionValidator` else it will be incompatible with Qt6.

Reply:
Done